app/controllers/spree/admin/payments_controller.rb

```diff
@@ -33,20 +33,12 @@ def create
             return
           end
 
-          if @order.completed?
-            authorize_stripe_sca_payment
-            @payment.process_offline!
-            flash[:success] = flash_message_for(@payment, :successfully_created)
+          OrderWorkflow.new(@order).complete! unless @order.completed?
 
-            redirect_to spree.admin_order_payments_path(@order)
-          else
-            OrderWorkflow.new(@order).complete!
-            authorize_stripe_sca_payment
-            @payment.process_offline!
-
-            flash[:success] = Spree.t(:new_order_completed)
-            redirect_to spree.edit_admin_order_url(@order)
```

Comment:
The frontend was redirecting to the index. This change ensures the backend always returns the index in a successful scenario.

Reply:
Hmm, so this "new order completed" code path was essentially dead code? Assuming this was the only form that submits here, I guess it hasn't been used in at least a few years. 
In any case, we certainly don't seem to have specs covering it. 


Therefore can you please also remove the now-unused [translation key](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/locales/en.yml#L3784)?